writer/writer.go

```diff
@@ -300,8 +302,10 @@ func (pw *ParquetWriter) flushObjs() error {
 						}()
 
 					} else {
-						pagesMapList[index][name], _ = layout.TableToDataPages(table, int32(pw.PageSize),
-							pw.CompressionType)
+						if table.Schema.Type != nil {
+							pagesMapList[index][name], _ = layout.TableToDataPages(table, int32(pw.PageSize),
+								pw.CompressionType)
+						}
```

Comment:
Tried to wrap the invocation with a nil guard, and that seems to have helped. 

Ran the test suite and it looks fine.

Is this correct? Am I missing something?